### PR TITLE
Make credit_specification an object and change cpu_credits to a string

### PR DESCRIPTION
**Pull request by djk**
I was trying to disable T3 unlimited but ran into errors where it was expecting a list, and when I gave it a list it expected an object. Had a look and saw that it wasn't quite right, so have corrected it.

### CODE REVIEW

**Comment by djk** [comment]
Yes, I patched it myself then used it then thought I should contribute the fix! Works as expected.

File: variables.tf [hunk 1]
```diff
@@ -126,8 +126,8 @@ variable "placement" {
 
 variable "credit_specification" {
```
Comment (aknysh) [inline variables.tf]:
```suggestion
variable "cpu_credits" {
```

for consistency, let's name it the same as the module's attribute

File: main.tf [hunk 2]
```diff
@@ -26,6 +25,10 @@ resource "aws_launch_template" "default" {
   placement                            = ["${var.placement}"]
   user_data                            = "${var.user_data_base64}"
 
+  credit_specification {
+    cpu_credits = "${var.credit_specification}"
```
Comment (aknysh) [inline main.tf]:
```suggestion
    cpu_credits = "${var. cpu_credits}"
```

File: variables.tf (hunk 1 above)
Comment (djk) [reply variables.tf]: Done!

**Pull request closed without merging**